### ISSUE TITLE
When running copyWith, not all types are full move the the copyWith method

### ISSUE THREAD

**Issue by Luckey-Elijah** [AdrienAudouard/flutter-utils#4]
I have the code:
```dart
class CommandContext<Result> {
  CommandContext({
    required this.invocation,
    required this.rest,
    required this.current,
    required this.result,
  });

  final List<String> invocation;
  final List<String> rest;
  final Command<Result> current;
  final Result result;
}
```
And when I execute _Generate copyWith_, this is generated in the above class.
```dart
CommandContext copyWith({
  List? invocation,
  List? rest,
  Command? current,
  Result? result,
}) {
  return CommandContext(
    invocation: invocation ?? this.invocation,
    rest: rest ?? this.rest,
    current: current ?? this.current,
    result: result ?? this.result,
  );
}
```
It is missing the type in the `<>` for `invocation`, `rest`, and `current`

**Comment by AdrienAudouard** [comment]
Indeed there is a problem, I'm gonna fix that soon.

**Comment by AdrienAudouard** [comment]
It has been fixed in the `2.1.1` released this morning. 
Thanks for your help 🙏

**Comment by Luckey-Elijah** [comment]
Thank you!
